Update README.md (#316)
mattermost/mattermost-handbook: company/how-to-guides-for-staff/how-to-spend-company-money/README.md

# How to spend company money

At Mattermost, we want you to be successful from Day 1. As staff members, you're empowered to determine and purchase what you need in order to do your job. We value high impact over stringent expense rules and these expense guidelines are intended to help make decisions not hinder movement. Depending on country/region, some items may be way below or way above the averages listed. If you need to spend more to get the right equipment for you to be productive, we trust you’re making purchase decisions to optimize cost and efficiency. 

> If you're uncertain, feel free to reach out in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel.

Reviewer: ```suggestion
If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or direct message @isabel.alorro on Mattermost.
```

Reviewer: ^although asking questions in public is encouraged, some may feel more comfortable reaching out directly, especially if they're new staff members

Author: +1, definitely agree.

> If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or direct message @isabel.alorro on Mattermost.

Author: ```suggestion
If you're uncertain, feel free to ask in the [Finance Team](https://community.mattermost.com/private-core/channels/expenses) channel, or reach out to expenses@mattermost.com.
```

* **PC:** Any suitable laptop with comparable specifications and lower or similar cost to the recommended MacBook. Many of our engineers use Lenovo laptops from the Thinkpad X, T, or P series. If in doubt, we recommend a Lenovo Thinkpad T series.

### MOBILE DEVICE POLICY
### Mobile device policy

The Mattermost mobile phone policy enables the use of mobile devices for customer-facing roles or those providing repeating on-call service in a secure fashion.
The Mattermost mobile device policy enables the use of mobile devices for customer-facing roles or those providing repeating on-call service in a secure fashion.

#### Subscription

If you are using your phone primarily for business related matters as defined by your local tax laws, we will reimburse up to 100 USD. Positions or roles where mobile reimbursement may be needed include the following:
If you are using your device primarily for business-related matters as defined by your local tax laws, we will reimburse up to 100 USD. Positions or roles where mobile reimbursement may be needed include the following:

* Customer-facing roles: 

>     * Account Executives

Reviewer: I believe we do still have AEs as customer-facing role

If there is a piece of software you are looking to buy that Mattermost does not have a subscription to, discuss with your manager and then post in the [Systems channel](https://community.mattermost.com/private-core/channels/systems) to discuss with the Procurement team.  

Note that any software expensed that is not been vetted through Procurement will not be reimbursed. Should you have questions, reach out to the [Purchases](https://community.mattermost.com/private-core/channels/purchases) channel.

## Travel

Mattermost will cover work-related travel expenses for approved business travel. You may extend a business trip for personal reasons, but you will only be expensed for meals and lodging that are part of the business-related portion of the trip. Use the same Expense Guidelines when spending on air, lodging, and meals. 

### Meals and entertainment

Spend company money like it’s your own. When traveling on behalf of Mattermost, average meal costs will vary by destination city and country. The two links below are helpful reference points for city and country averages. Note: 20% is considered the maximum gratuity for exceptional service. 

> The below links provide information regarding daily per diem guidance in the US as well as other countries.

Reviewer: ```suggestion
The below links provide a few examples regarding daily per diem guidance.
```
